client/js/views/list_view.js

```diff
@@ -70,6 +70,11 @@ App.ListView = Backbone.View.extend({
             this.model.collection.board.cards.bind('change:is_archived', this.renderCardsCollection);
             this.model.collection.board.cards.bind('change:comment_count', this.renderCardsCollection);
             this.model.collection.board.cards.bind('change:list_id', this.renderCardsCollection);
+            this.model.collection.board.cards.bind('change:is_filtered', function(e){
+                // TODO not synchronized models here?
+                console.log(e.get('is_filtered'));
```

Comment:
@rrjanbiah Do you know why in the list_view context the cards are not really in sync with other components?
For example the same construct is in card_view and there the attribute is correctly applied.

Reply:
Always instantiating jQuery Selectors is slow (see http://vanilla-js.com/ or https://jsperf.com/).
The big problem of jQuery usage is generating spagetti code (for example filtering in card_view and header).

So i am not sure if there are performance benefits in using jquery for manipulation of the DOM.

Reply:
Thanks for your feedback.

Why do we have to render all cards even if they would be filtered? In this case i would implement the render function to skip the rendering of the card. As a bonus (compared to the current jQuery implementation) we would reduce the size of the DOM and increase performance where we do not need all cards shown.

I really think that it doesn't affect the performance at all and we could test that before using it in production.
My question over a month ago was the problem of outsynched models. This means that changing values in the card model does not really affect the card models in the list model (and vice-versa). 
Do you decoupled the model structures somewhere from each other? From my understanding an used model should be synched through the whole web app.

Reply:
In `board_header_view.js` line 1841 `cardFilter function()`, you have set `is_filtered` attributes for the `cards  in the board model` 

In `list_view.js` line 1187 `renderCardNumbers function()`,  you have `filtered the cards from list model`.

 If you want to filter the cards from the board model 
In  `list_view.js` `renderCardNumbers function()`
- In line 1186 add `var self = this;`
- In line 1187  change it from 
`var filteredElements = this.model.cards.filter(function(card) {`  to  `var filteredElements = this.model.collection.board.cards.filter(function(card) {`  and please provide feedback after tried this change of code
- In line 1188 change it from
` return card.get('is_archived') !== 1 && card.get('is_filtered') === false;`  to  `return card.get('is_archived') !== 1 && card.get('list_id') === parseInt(self.model.attributes.id) && card.get('is_filtered') === false;`

please provide feedback after tried this change of codes

Reply:
Thx seems to work for triggering the rerendering of card numbers after filter application.

Currently i do not have the time (and priority) to further develop on the counter for the rows. Maybe if you want to can continue on the code base or stay/learn from ideas for the future restructuring of the client.

Reply:
Thanks for your contribution, we will continue the code change and will  merge the changes